### DISCUSSION TITLE
Code is incorrect. Please try again.

### DISCUSSION THREAD

**Discussion by ONE-Harmonaut** [polymorpher/one-wallet#177] [General]
I am trying to create a 1Wallet but keep receiving this error: Code is incorrect. Please try again.. I hit create wallet, scan QR with Google Auth app and I get this Code is incorrect. Please try again.. Any body else having this issue? I tried on different devices and in incognito mode. Thanks in advance.

**Comment by polymorpher** [comment]
Most likely your computer or your phone's clocks are out of sync. In this step, the client simply generates an expected code using your system's clock and compare with the code you typed in.
